design/decisions/why-sql/index.md

```diff
@@ -1,16 +1,194 @@
 ---
-title: "Why SQLite (or other SQL variant)"
-# description: ""
-author: "Kristiane Beicher"
-# date: ""
-draft: true
+author: Kristiane Beicher
 categories:
-  - backend
-  - database
-  - software
-  - tools
-  - technology
+- backend
+- database
+- software
+- tools
+- technology
+draft: true
+title: Why PostgreSQL
 ---
 
-Rationale for why we are choosing the SQL variant, based on guiding
-principles in `design/overview.md`
+# Backend technology
+
+We will at some point be selecting an open source database as the
+backbone of our project. This will support not only the access and
```

Comment:
```suggestion
An open source database is the
backbone of our project. It will support the access,
```

Don't need to talk in the present about the future. You can write more in the affirmative.